pyquil/gates.py

```diff
@@ -773,29 +774,29 @@ def GE(classical_reg1, classical_reg2, classical_reg3):
     return ClassicalGreaterEqual(classical_reg1, classical_reg2, classical_reg3)
 
 
-QUANTUM_GATES = {'I': I,
-                 'X': X,
-                 'Y': Y,
-                 'Z': Z,
-                 'H': H,
-                 'S': S,
-                 'T': T,
-                 'PHASE': PHASE,
-                 'RX': RX,
-                 'RY': RY,
-                 'RZ': RZ,
-                 'CZ': CZ,
-                 'CNOT': CNOT,
-                 'CCNOT': CCNOT,
-                 'CPHASE00': CPHASE00,
-                 'CPHASE01': CPHASE01,
-                 'CPHASE10': CPHASE10,
-                 'CPHASE': CPHASE,
-                 'SWAP': SWAP,
-                 'CSWAP': CSWAP,
-                 'ISWAP': ISWAP,
-                 'PSWAP': PSWAP
-                 }
+QUANTUM_GATES: Dict[str, Callable[..., Gate]] = {'I': I,
```

Comment:
would somewhat strongly prefer a line break after the opening brace `{` and then all key value pairs at a single tab indentation level.